### PR TITLE
Documentation for cp_package tag

### PR DESCRIPTION
**Pull request by wadhwa1**
# Description

Added documentation for cp_package tag.

Fixes # (issue)
https://confluentinc.atlassian.net/browse/ANSIENG-1960

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

**Review by anuj-apdev — APPROVED** [review]
LGTM

**Review by utkarsh5474 — APPROVED** [review]
Thanks, we can also add it to the prev PR

File: docs/TAGS.md [hunk 1]
```diff
@@ -93,7 +93,13 @@ Description: To generate secrets protection masterkey if secrets protection enab
 
 ### Tag - package
 
-Description: This is applied to all package installation related tasks. Skip this if you don't want to reinstall.
+Description: This is applied for all common packages installation tasks [like Java, openssl, gnupg, pip, etc]. Skip this if you don't want to reinstall.
```
Comment (nsharma-git) [inline docs/TAGS.md:96]: packages would be superset of all common and CP packages

**Review by nsharma-git — CHANGES_REQUESTED** [review]

**Comment by wadhwa1** [comment]
> Thanks, we can also add it to the prev PR

Thanks @utkarsh5474 , we actually have TAGS.md doc from 7.2.x branch onwards, hence created here.

File: docs/TAGS.md (hunk 1 above)
Comment (wadhwa1) [reply docs/TAGS.md]: yes, thanks @nsharma-git , re-worded the doc.

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**